CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
   consists of `Fullscreen::Exclusive(VideoMode)` and
   `Fullscreen::Borderless(MonitorHandle)` variants.
     - Adds support for exclusive fullscreen mode.
+- On iOS add support for hiding the home indicator.
+- On iOS add support for deferring system gestures.
 
```

Comment:
It would be good to mention the monitor name bug fix too. At the very least, it'll make iOS seem more active.

Reply:
(Also, to be bikesheddy, it's conventional to put a comma after the platform: "On iOS, add...")